includes/class-fs-plugin-updater.php

```diff
@@ -186,7 +191,12 @@ function pre_set_site_transient_update_plugins_filter( $transient_data ) {
 					$plugin_details->new_version = $new_version->version;
 					$plugin_details->url         = WP_FS__ADDRESS;
 					$plugin_details->package     = $new_version->url;
-					$plugin_details->plugin      = $this->_fs->get_plugin_basename();
+
+					if ( $this->_fs->get_plugin()->is_plugin() ) {
```

Comment:
@fajardoleo I suggest to add `is_plugin()` as a helper method in `Freemius` and use it instead to make the code cleaner.